Client/Client.csproj

```diff
@@ -2,11 +2,13 @@
 
     <PropertyGroup>
         <OutputType>Exe</OutputType>
-        <TargetFramework>netcoreapp2.0</TargetFramework>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Let me share one experience about this version. I used brew to install dotnetcore and then when I run `dotnet run --project ClientWithSsl`, it errored with `netcoreapp2.0 is not found, I have only 5.0`. So, is `netcoreapp3.1` up-to-date version? if so, we need to provide an information in UI since most probably it will fail on customer's machine. They tend to run this project right away in  terminal with commands we provide in Cloud Console.

Reply:
3.1 is the current LTS version of .NET, and 5.0 is the latest version
5.0 should compile 3.1 projects without problems (would be interested to know if, on your machine, just changing that `TargetFramework` allows the solution to build?)

Reply:
I think this 3.1 is ok we can somehow  state in Console no problems